Development/src/pages/devices/DevicesShow.js

```diff
@@ -32,12 +40,72 @@ export const DevicesShow = props => {
 };
 
 const DevicesShowView = props => {
+    const { record } = useRecordContext();
+    const [useIS08API, setUseIS08API] = useState(false);
```

Comment:
Although it would be more verbose, I'd prefer `useChannelMappingAPI`, since AMWA IS-xx specs sometimes define multiple APIs.